src/consensus/forging.md

```diff
@@ -0,0 +1,84 @@
+# Forging new blocks
+
+Forging new blocks is the process of packing data that has not yet been included in the
+blockchain (pending transactions) into a fresh block. Blocks are signed by stake pool
+credentials, so only the nodes which are configured with such credentials strictly need to
```

Comment:
I'm still puzzled by the use of "strictly need to". I would say these nodes are the only nodes that can forge blocks.

Reply:
What about 
> only the nodes which are configured with such credentials would be able to forge blocks and therefore only those are strictly need to...